### ISSUE TITLE
API delegering - Makinporten link doesnt open in a new tab

### ISSUE THREAD

**Issue by anishastphen** [Altinn/altinn-access-management#408]

**Comment by anishastphen** [comment]
@jonkjetiloye @rtafj can you guys evaluate the need for fix here?

**Comment by jonkjetiloye** [comment]
@anishastphen 
I agree, this should open in a new tab to not break the users workflow and have to do browser navigation in order to get back and continue.

@Albertlarsen 
Not critical for go live but should be a quick little fix I guess
